### PR TITLE
Feat/global styles/native

### PR DESCRIPTION
**Pull request by lee-cheung**
global styles for colors, text and layout

### CODE REVIEW

File: NativeApp/src/utilities/globalStyles/colours.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
-const colours = {
+const colors = {
```
Comment (kirsty-unosquare) [inline NativeApp/src/utilities/globalStyles/colours.js]: Don't need this file anymore. Can delete along with the globalStyles folder.

File: NativeApp/src/utilities/globalStyles/text.js [hunk 2]
```diff
@@ -1,6 +0,0 @@
-export const P_SIZE = 17;
```
Comment (kirsty-unosquare) [inline NativeApp/src/utilities/globalStyles/text.js:1]: Same here. Can be deleted.

**Review by kirsty-unosquare — CHANGES_REQUESTED** [review]
Forgot to remove the globalStyles folder. Other than that all good. @William-Unosquare just needs to have a quick look over it. :)

File: NativeApp/src/utilities/globalStyles/colours.js (hunk 1 above)
Comment (lee-cheung) [reply NativeApp/src/utilities/globalStyles/colours.js]: woops, yes I will delete it

**Comment by kirsty-unosquare** [comment]
Gonna approve.

**Review by kirsty-unosquare — APPROVED** [review]

**Review by William-Unosquare — APPROVED** [review]
Looks good. It's a bit hard reviewing some of these bigger PRs without being involved in the project though so stuff could be missed.

**Pull request merged**